tests/Kernel/LanguageSwitcherTest.php

```diff
@@ -40,30 +26,115 @@ public function testLanguageSwitcherRendering(): void {
     $actual = $crawler->filter('.ecl-dialog .ecl-dialog__title')->text();
     $this->assertEquals('Select your language', trim($actual));
 
-    // Make sure that language switcher link is properly rendered.
-    $actual = $crawler->filter('a.ecl-lang-select-sites__link > .ecl-lang-select-sites__label')->text();
-    $this->assertEquals('English', $actual);
-
-    $actual = $crawler->filter('a.ecl-lang-select-sites__link > .ecl-lang-select-sites__code > .ecl-lang-select-sites__code-text')->text();
-    $this->assertEquals('en', $actual);
-
     /** @var \Drupal\Core\Language\LanguageInterface[] $languages */
-    $languages = \Drupal::service('language_manager')->getNativeLanguages();
+    $languages = $this->container->get('language_manager')->getNativeLanguages();
+
+    $lang_config = \Drupal::configFactory()->getEditable('language.negotiation');
```

Comment:
If you use `$this->container` for the language manager, let's use for all services no?

Also, why `getEditable()` ?